x-pack/plugins/uptime/public/components/overview/alerts/monitor_expressions/status_expression_select.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiCheckbox } from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+import { DownNoExpressionSelect } from './down_number_select';
+import { TimeExpressionSelect } from './time_expression_select';
+import { statusExpLabels } from './translations';
+
+interface Props {
+  alertParams: { [param: string]: any };
+  hasFilters: boolean;
+  setAlertParams: (key: string, value: any) => void;
+}
+
+export const StatusExpressionSelect: React.FC<Props> = ({
+  alertParams,
+  hasFilters,
+  setAlertParams,
+}) => {
+  const [isEnabled, setIsEnabled] = useState<boolean>(true);
+
+  useEffect(() => {
+    setAlertParams('shouldCheckStatus', isEnabled);
+  }, [isEnabled, setAlertParams]);
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="s">
+      <EuiFlexItem>
+        <EuiCheckbox
+          id="statusEnabled"
+          label={statusExpLabels.ENABLED_CHECKBOX}
+          checked={isEnabled}
+          onChange={() => setIsEnabled(!isEnabled)}
```

Comment:
Something is wonky here, deselecting this box and saving it doesn't persist this being disabled for me.

Reply:
Well-spotted; this should be working now. Fixed in https://github.com/elastic/kibana/pull/70284/commits/bd4b3b3af1156bd051fde89bc2651c88cf10a215.